internal/runners/invite/invite.go

```diff
@@ -63,7 +64,9 @@ func (i *invite) Run(params *Params) error {
 		}
 	}
 
-	emails := strings.Split(params.EmailList, ",")
+	re := regexp.MustCompile(",,+")
```

Comment:
Nit: even though you only use the regex on the next line it might be nice to name it something more descriptive

Reply:
That's fair. I will change it.